File: trabajos en javascript/condicionales.js
var prompt = require('prompt-sync')();

var figura = prompt("Ingrese la figura a calcular: cuadrado, triangulo o circulo: ");
var pi = 3.1416;


if (figura == "cuadrado") {
    var lado = prompt("Ingrese el lado del cuadrado");
    var area = lado * lado;
    console.log("El area del cuadrado es: " + area);
}
else if (figura == "triangulo") {
    var base = prompt("Ingrese la base del triangulo");
    var altura = prompt("Ingrese la altura del triangulo");
    var area = (base * altura) / 2;
    console.log("El area del triangulo es: " + area);
}
else if (figura == "circulo") {
    var radio = prompt("Ingrese el radio del circulo");
    var area = pi * radio * radio;
    console.log("El area del circulo es: " + area);
}
else {
    console.log("La figura ingresada no es valida");
}

File: trabajos en rubi/condicionales.rb
print "ingresa la figura que desea calcular: 0 para cuadrado, 1 para triangulo, 2 para circulo: "
figura = gets.chomp.to_i

if figura == 0
    puts "ingresa el lado del cuadrado: "
    lado = gets.chomp.to_i
    area = lado * lado
    puts "el area del cuadrado es: #{area}"
elsif figura == 1
    puts "ingresa la base del triangulo: "
    base = gets.chomp.to_i
    puts "ingresa la altura del triangulo: "
    altura = gets.chomp.to_i
    area = (base * altura) / 2
    puts "el area del triangulo es: #{area}"
elsif figura == 2

    puts "ingresa el radio del circulo: "
    radio = gets.chomp.to_i
    area = 3.1416 * radio * radio
    puts "el area del circulo es: #{area}"
else
    puts "ingresa una opcion valida"
end
print ("fin de la ejecucion_ escriba exit para salir:")
Z = gets.chomp.to_i